crates/precompile/src/blob/kzg_settings/mod.rs

```diff
@@ -0,0 +1,33 @@
+use c_kzg::KzgSettings;
```

Comment:
This module is exported, lmk wyt @Rjected @mattsse

Reply:
I tend not to use`mod` inside `revm`

Reply:
With `EnvCfg` path of setting kzg setting, this is unnecessary.